src/ontology/mondo-ingest.Makefile

```diff
@@ -549,23 +549,24 @@ $(EXTERNAL_CONTENT_DIR)/%.robot.owl: $(EXTERNAL_CONTENT_DIR)/%.robot.tsv
 $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv: $(TMPDIR)/nord.tsv config/external-content-robot-headers.json
 	mkdir -p $(EXTERNAL_CONTENT_DIR)
 	python ../scripts/add-robot-template-header.py $^ > $@
-.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
+.PRECIOUS: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv
 
 .PHONY: external-content-nord
-external-content-nord: $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
+external-content-nord: $(EXTERNAL_CONTENT_DIR)/nord.robot.tsv $(EXTERNAL_CONTENT_DIR)/nord.robot.owl
 
 tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
 	$(ROBOT) query -i $< --query ../sparql/select-ordo-subsets.sparql $@
 
-$(EXTERNAL_CONTENT_DIR)/ordo-subsets.robot.template.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
+$(EXTERNAL_CONTENT_DIR)/ordo-subsets.robot.tsv: tmp/ordo-subsets.tsv tmp/mondo.sssom.tsv
```

Comment:
Renamed the file so it is the same with the NORD one. this is necessary so that the OWL version can be automatically generated, see row 541 in the makefile to see how.

Reply:
Makes sense!
I'm going to merge this, but I do have some comments, not related to any particular change, so I'm going to add them to the [original discussion](https://github.com/monarch-initiative/mondo-ingest/pull/543#discussion_r1615036198) in the PR we're merging this into.